Octokit/Models/Request/MergePullRequest.cs

```diff
@@ -11,23 +11,21 @@ namespace Octokit
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
     public class MergePullRequest
     {
-        public MergePullRequest(string message)
-        {
-            Ensure.ArgumentNotNull(message, "message");
-
-            Message = message;
-        }
-
         /// <summary>
         /// The message that will be used for the merge commit (optional)
         /// </summary>
-        public string Message { get; private set; }
+        public string Message { get; set; }
```

Comment:
Well the _response_ is `message` but the _request_ is `commit_message`. The docs are actually a bit unclear on this so I'll ask @pengwynn for clarification.